datasets/huishoudelijkafval/huishoudelijkafval.json

```diff
@@ -638,6 +638,224 @@
+          },
+          "loopafstandCategorieTot": {
+            "type": "integer",
+            "provenance": "loopafstand_categorie_tot",
+            "description": "Bovengrens van de categorie waarin de berekende loopafstand valt"
+          },
+          "wijzigingsdatumDp": {
+            "type": "string",
+            "provenance": "wijzigingsdatum_dp",
+            "format": "date-time",
+            "description": "Datum waarop het object is gewijzigd"
+          },
+          "verwijderdDp": {
+            "type": "boolean",
+            "provenance": "verwijderd_dp",
+            "description": "indicatie of het object verwijderd is"
+          }
+        }
+      }
+    },
+    {
+      "id": "bag_object_loopafstand",
+      "type": "table",
+      "provenance": "afval_bag_object_loopafstand ",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["id", "schema"],
+        "display": "id",
+        "properties": {
+          "id": {
+            "type": "integer"
+          },
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "bagObjectId": {
+            "type": "string",
+            "provenance": "bag_object_id",
+            "description": "Uniek identificerend kenmerk van bagobject"
+          },
+          "bagObjectType": {
+            "type": "string",
+            "provenance": "bag_object_type",
+            "description": "BAG objectklasse: pand, ligplaats, standplaats"
+          },
+          "clusterId": {
+            "type": "string",
+            "provenance": "cluster_id",
+            "description": "Uniek identificerend kenmerk van cluster waarnaar de afstand is berekend"
+          },
+          "fractie": {
+            "type": "string",
+            "provenance": "afval_fractie",
+            "description": "Code afvalfractie waarnaar de afstand is bepaald"
+          },
+          "fractieOmschrijving": {
+            "type": "string",
+            "provenance": "afval_fractie_omschrijving",
+            "description": "Omschrijving afvalfractie waarnaar de afstand is bepaald."
+          },
+          "loopafstandCategorieId": {
+            "type": "string",
+            "provenance": "afval_loopafstand_categorie_id",
+            "relation": "huishoudelijkafval:loopafstand_categorie",
+            "description": "relatie naar de categorie waarin de berekende loopafstand valt"
+          },
+          "loopafstand": {
+            "type": "numeric",
+            "provenance": "loopafstand",
+            "description": "Berekende afstand over de weg in meter in RD (epsg:28992) van woning tot cluster van containers van de betreffende afvalfractie"
+          },
+          "geometrie": {
+            "type": "geometry",
+            "provenance": "geometry",
```

Comment:
Klopt deze provenance wel? In alle overige situaties heet het veld "geometrie" in de huishoudelijkafval tabellen.